app/controllers/public_reviews_controller.rb

```diff
@@ -26,7 +26,55 @@ def finish
     flash[:success] = 'Dziękujemy za zgłoszenie'
   end
 
+  def new
```

Comment:
Ponadto przy metodzie `update` należy wziąć pod uwagę wymagania opisane w zadaniu - przede wszystkim to, że użytkownik wprowadza swój e-mail. Jest to prosta forma weryfikacji, czy nieupoważniona osoba nie wprowadza recenzji.